### ISSUE TITLE
Make resizable a Javadoc popup hint window

### ISSUE THREAD

**Issue by bykka** [apache/netbeans#5058]
### Description

A Javadoc popup hint window has a fixed size and for cases of big javadoc descriptions, it requires to do a lot of scrolling.

### Use case/motivation

Would be nice to make it resizable like in eclipse.
[ice_video_20221207-205605.webm](https://user-images.githubusercontent.com/489997/206282819-7873b08b-934e-4d82-8dbc-45ac48374c78.webm)

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by mbien** [comment]
what should happen if you:

1. resize window
2. close
3. and open it again?

**Comment by mbien** [comment]
agreed. Persistence is probably not necessary for this (or could be still added later if it turns out to be).

**Comment by bykka** [comment]
What module is responsible for the Javadoc popup window?

**Comment by mbien** [comment]
should be in `ide/editor.completion`
you can set a breakpoint at
https://github.com/apache/netbeans/blob/cf48b1fd9a97a8294c10b99fa00ad488f5837206/ide/editor.completion/src/org/netbeans/modules/editor/completion/CompletionLayout.java#L502
and you should be in the general area.

**Comment by rothardo** [comment]
@mbien is this issue stil open?
I would like to work on issues like this please assign me a task if possible, I am new to open source
